tests/SerializerAbstractTest.php

```diff
@@ -409,6 +420,24 @@ public function testSetAllObjectSerialize()
         $serializer->setAllObjectSerialize(false);
         $this->assertFalse($serializer->getAllObjectSerialize());
     }
+
+    public function testClippingUTF8Characters()
+    {
+        if (!extension_loaded('mbstring')) {
+            $this->markTestSkipped('mbstring extension is not enabled.');
+        }
+
+        $testString = 'Прекратите надеяться, что ваши пользователи будут сообщать об ошибках';
```

Comment:
This is so funny! It's the Google-translated version of `Stop hoping that your users will report bugs` in russian :trollface:

Reply:
I couldn't just add some random text, needs to make a little bit sense at least 😉